pkg/okteto/auth.go

```diff
@@ -163,7 +165,27 @@ func GetURL() string {
 	return t.URL
 }
 
-func saveToken(id, token, url string) error {
+// GetRegistry returns the URL of the registry
+func GetRegistry() string {
+	t, err := GetToken()
+	if err != nil {
+		return "registry.cloud.okteto.net"
+	}
+
+	return t.Registry
+}
+
+// GetBuildkit returns the URL of the authenticated user
+func GetBuildkit() string {
+	t, err := GetToken()
+	if err != nil {
+		return "registry.cloud.okteto.net"
```

Comment:
registry -> buildkit

Reply:
I would return and error here, auth will fail otherwise

Reply:
The logic should be return error if no okteto login. Return this default value  if it is empty (needs to login again)